### ISSUE TITLE
show the real-time progress of current run

### ISSUE THREAD

**Issue by VinodKumar9576** [lucafaggianelli/plombery#391]
During the current run, the app doesn't work and giving error, instead can we have the progress of it in real time, the app & the log shows only after the current run is completed. It's difficult to track where the current run is if the job takes long time to run.

<img width="506" alt="image" src="https://github.com/lucafaggianelli/plombery/assets/23068132/3aa2bc83-e15b-4aa1-a865-040c9915e7c9">

**Comment by D3m0n1cC09n1710n** [comment]
Yeah its the same issue as #149 IMO - non-async tasks doent cause this issue.
